### PR TITLE
Add Github Action details to CI.md

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

We have the CI jobs from Zuul to GithubActions. This PR adds the documentation related to CI.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by tremble — COMMENTED** [review]
Generally looks good, some minor musings:
- Might be good to link to descriptions of the tests
- I'd split out the different linters (makes adding links easier)
- We should using sanity tests against 3.6 and 3.7 for at least 6.x

**Comment by GomathiselviS** [comment]
> Generally looks good, some minor musings:
> 
> * Might be good to link to descriptions of the tests
> * I'd split out the different linters (makes adding links easier)
> * We should using sanity tests against 3.6 and 3.7 for at least 6.x

@tremble 

Regarding the sanity tests against py 3.6 and py 3.7 :
From [this document](https://github.com/ansible-collections/amazon.aws/pull/1519#pullrequestreview-1413703616), I see that from ansible 2.12 there is no support for py 3.7 and 3.7 on the controller node. Please correct me if my understanding is wrong.

Regarding the linters, I am waiting for a decision on [this comment](https://github.com/ansible-collections/amazon.aws/pull/1499#discussion_r1183509977). For now, I do not see any linters config added to tox.ini. We can add black and flake8 to tox.ini. Let me know what you think.

File: CI.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## AWS Collections
+
+GitHub Actions are used to run the Continuous Integration for amazon.aws collection. The workflows used for the CI can be found [here](https://github.com/ansible-collections/amazon.aws/tree/main/.github/workflows). These workflows include jobs to run the unit tests, integration tests, sanity tests, linters, changelog check and doc related checks. The following table lists the python and ansible versions against which these jobs are run.
+
+| Jobs | Description | Python Versions | Ansible Versions |
+| ------ |-------| ------ | -----------|
+| changelog |Checks for the presence of Changelog fragments | 3.9 | devel |
+| Linters | Runs `black` and `flake8` on plugins and tests | 3.9 | devel |
+| Sanity | Runs absible sanity checks | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
```
Comment (alinabuzachis) [inline CI.md:11]:
```suggestion
| Sanity | Runs ansible sanity checks | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
```

**Review by alinabuzachis — COMMENTED** [review]
I guess we should also probably let know users how to contribute, where are the flow located or something like that

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**